src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponentSpec.scala

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.util.UUID
+
+import org.broadinstitute.dsde.rawls.model._
+import org.joda.time.DateTime
+
+/**
+ * Created by mbemis on 2/16/16.
+ */
+class MethodConfigurationComponentSpec extends TestDriverComponent with MethodConfigurationComponent with WorkspaceComponent {
```

Comment:
should I port the old test cases to this code as best as possible? or is this test class sufficient?